releasetasks/templates/mobile/candidates_fennec.yml.tmpl

```diff
@@ -5,6 +5,7 @@
     task:
         provisionerId: aws-provisioner-v1
         workerType: gecko-decision
+        schedulerId: gecko-level-3
```

Comment:
why is this needed?

Reply:
This failure occurred without it: https://tools.taskcluster.net/groups/ffYqzngiQUiOFEdlZ_8bVQ/tasks/ffYqzngiQUiOFEdlZ_8bVQ/details

Basically, TC checks that the same scheduler schedules all the task of a taskgroup, which is not really true here :?

Reply:
Ah, right. Thanks for explanations!